### ISSUE TITLE
clojure.spec?

### ISSUE THREAD

**Issue by devn** [SparkFund/spec-tacular#27]
Any thoughts on leveraging clojure.spec in the future?

**Comment by calvis** [comment]
Yes, although I'm not entirely sure what integration would be most useful.  Considering generating clojure.spec based on defspec or generating defspecs based on existing clojure.specs.  But I think since clojure.spec's entity maps are based on namespaced keywords, the translation might not be absolutely straightforward.
